src/data-table/data-table.js

```diff
@@ -587,24 +570,63 @@ export function Unstable_DataTable(props: DataTablePropsT) {
     [props.onSort],
   );
 
+  const [columnHighlightIndex, setColumnHighlightIndex] = React.useState(-1);
+  const [rowHighlightIndex, setRowHighlightIndex] = React.useState(-1);
+
+  function handleRowHighlightIndexChange(nextIndex) {
+    setRowHighlightIndex(nextIndex);
+    if (gridRef.current) {
+      if (nextIndex >= 0) {
+        // $FlowFixMe - unable to get react-window types
+        gridRef.current.scrollToItem({rowIndex: nextIndex});
```

Comment:
i think this can cause a slight jump when mousing over a partially shown row.

![rowslightjump](https://user-images.githubusercontent.com/1939257/71131444-9323ad80-21a9-11ea-9660-e8ebde61de33.gif)

Reply:
this is a 'feature' of react-window, i'm afraid i can't turn this off

Reply:
https://github.com/bvaughn/react-window/blob/51a24c4972c9ca5eee4df5ea3ec25bc4328e40c5/src/__tests__/__snapshots__/VariableSizeGrid.js.snap#L92

Reply:
Ah okay - not a big issue.